client/src/components/CommentsUser.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+export default class CommentsUser extends React.Component {
+  constructor() {
+    super();
+  }
+  render() {
+    let userStars;
+    if (this.props.data.stars === 1) {
+      userStars = "oneStar";
+    } else if (this.props.data.stars === 2) {
+      userStars = "twoStar";
+    } else if (this.props.data.stars === 3) {
+      userStars = "threeStar";
+    } else if (this.props.data.stars === 4) {
+      userStars = "fourStar";
+    } else {
+      userStars = "fiveStar ";
+    }
+    return (
+      <div className="reviewSection">
+        <div className="starandtime">
+          <div className={userStars} />
+          <div className="dateWritten">{this.props.data.date}</div>
+        </div>
+        <p className="userText">{this.props.data.comments}</p>
+      </div>
+    );
+  }
+}
```

Comment:
Ran out of time to go through the rest of this PR. Will pick back up tomorrow, or when I get home. Looks good so far!